README.md

```diff
@@ -4,7 +4,7 @@ This repository contains Cert Manager Operator designed for OpenShift. The opera
 
 ## The operator architecture and design assumptions
 
-The Operator uses the [upstream deployment manifests](https://github.com/jetstack/cert-manager/releases/download/v1.15.5/cert-manager.yaml). It divides them into separate files and deploys using 3 controllers:
+The Operator uses the [upstream deployment manifests](https://github.com/jetstack/cert-manager/releases/download/v1.16.4/cert-manager.yaml). It divides them into separate files and deploys using 3 controllers:
```

Comment:
If we make this link generic and update it to `https://github.com/cert-manager/cert-manager/releases` so we don't need to update this link during every rebase manually.